commands/server/init.go

```diff
@@ -17,9 +18,13 @@ import (
 const (
 	// AppStateKey is the key in the json json where all info
 	// on initializing the app can be found
-	AppStateKey   = "app_state"
-	flagIndexAll  = "all"
-	flagIndexTags = "tags"
+	AppStateKey             = "app_state"
+	DirConfig               = "config"
+	GenesisTimeKey          = "genesis_time"
+	ErrorAlreadyInitialised = "the application has already been initialised, use %s flag to override"
+	FlagForce               = "f"
```

Comment:
You don't think this flag should be longer? like `-force`?

I guess `rm -f` is the same.... hmm....